### PR TITLE
Updates for AWS Exporter

### PR DESCRIPTION
**Pull request by jradhakrishnan**
[ch16008]

### CODE REVIEW

**Comment by arramos84** [comment]
is this assuming the node already has the role to scrape? What happens if someone sets to true? Is this ready to go with true set? Just want to make sure if it's officially release that it's supported. Comment above the `awsExporterEnabled` flag would be good. Maybe a link to our docs or some description as to how to use it.

**Comment by jradhakrishnan** [comment]
> is this assuming the node already has the role to scrape? What happens if someone sets to true? Is this ready to go with true set? Just want to make sure if it's officially release that it's supported. Comment above the `awsExporterEnabled` flag would be good. Maybe a link to our docs or some description as to how to use it.

This doesn't automatically trigger any of the AWS calls. One needs to add the AWS Account in the UI with the correct role. We have a documentation on how to add an account. The documentation needs to mention that `asserts-server` should be running with a role, and point the link to the CF template to create an account specific role with trust relationship with the source role.

**Review by arramos84 — APPROVED** [review]

File: charts/asserts/Chart.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Asserts Helm Chart to configure entire asserts stack
 icon: https://www.asserts.ai/favicon.png
 type: application
 
-version: 1.47.0
+version: 1.48.0
```
Comment (arramos84) [inline charts/asserts/Chart.yaml:7]: this version needs to be `1.49.0` now since we've merged another change.

**Review by arramos84 — CHANGES_REQUESTED** [review]
change version to `1.49.0`

**Comment by jradhakrishnan** [comment]
> change version to `1.49.0`

Done

**Review by arramos84 — APPROVED** [review]

**Pull request merged**